storybookjs/react-native
# feat: Re-enable the Boolean control (change it from a knob to a control)

Issue: #177

## What I did
Enabled the Boolean control, which was commented out.

## How to test

Please explain how to test your changes and consider the following questions

- Does this need a new example in examples/native? **Yes**. Added `BooleanExample.stories.tsx` and `BooleanExampleButton.tsx` for this (see below for discussion).
- Does this need an update to the documentation? **Not sure?**

## Discussion
The 6.0 controls and component story format are still a bit 🤯  to me, despite reading https://storybook.js.org/docs/react/essentials/controls and https://storybook.js.org/docs/react/api/csf . Thus, there might be things here I've simply misunderstood.

Another unclear part is - how do we want to structure the control examples? One  (or a couple) of large examples with multiple different controls? Or one example per control, e.g.

```
Controls
  Text
  Number
  Color
  Boolean
  ...
```

Also, added a couple of comments to this PR on the parts of the code which I'm unsure about.
